tests/e2e/e2e_gov_test.go

```diff
@@ -0,0 +1,296 @@
+package e2e
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"time"
+
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+)
+
+/*
+SendTokensFromNewGovAccount tests passing a gov proposal that sends tokens on behalf of the gov module to a recipient.
+Test Benchmarks:
+1. Subtest to fund the community pool via the distribution module
+2. Submission, deposit and vote of legacy proposal to fund the gov account from the community pool
+3. Validation that funds have been deposited to gov account
+4. Submission, deposit and vote of message based proposal to send funds from the gov account to a recipient account
+5. Validation that funds have been deposited to recipient account
+*/
+func (s *IntegrationTestSuite) SendTokensFromNewGovAccount() {
+	s.writeGovProposals(s.chainA)
```

Comment:
can we use a subtest?

```suggestion
	s.Run("description.....", func() {
		s.writeGovProposals(s.chainA)
		...
```